Kitchen Sync PR

rnystromtest opened the pull request:
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 

- Duis aute irure dolor in reprehenderit 
- in voluptate velit esse cillum dolore 
- eu fugiat nulla pariatur

> Excepteur sint occaecat cupidatat 

[non proident](http://google.com) sunt in **culpa** qui _officia_ deserunt ~~mollit~~ anim id est laborum.

1. one
2. two
3. three

rnystromtest committed:
edited file

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

rnystrom commented on .gitignore:
```diff
@@ -65,3 +65,5 @@ fastlane/report.xml
 fastlane/screenshots
 fastlane/test_output
+
+# new things bla bla bla
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 

- Duis aute irure dolor in reprehenderit 
- in voluptate velit esse cillum dolore 
- eu fugiat nulla pariatur

> Excepteur sint occaecat cupidatat 

[non proident](http://google.com) sunt in **culpa** qui _officia_ deserunt ~~mollit~~ anim id est laborum.

1. one
2. two
3. three

rnystromtest replied:
Testing a comment

rnystromtest replied:
Testing

rnystrom commented on .gitignore:
```diff
@@ -65,3 +65,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
```
Another comment

rnystromtest replied:
This is a new comment.

rnystrom commented on .gitignore:
```diff
@@ -65,3 +65,5 @@ fastlane/report.xml
 fastlane/screenshots
 fastlane/test_output
+
+# new things bla bla bla
```
and one more

rnystromtest replied:
Here is another comment that is a little longer with some autocomplete words. The new version of this is the first app that I’ve ever used to have a good time with the app.

- Item one
- Item two
- Item three

rnystrom replied:
Testing

rnystrom commented on .gitignore:
```diff
@@ -65,3 +65,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
+
```
one here too

rnystromtest commented:
@rnystrom test

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

rnystromtest commented:
1234 @rnystrom 

And another @rnystromtest **1234** and _italics_

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

rnystrom committed:
Update .gitignore

bla bla bla #34

rnystrom committed:
testing #34

https://github.com/rnystromtest/Githawk-Playground/pull/34

rnystromtest/Githawk-Playground#34

rnystrom committed:
Update .gitignore

Closes #34

rnystrom reviewed (commented):
test
